functions/src/firebase/env.ts

```diff
@@ -34,15 +35,23 @@ export const getTwilioEnv = (): TwilioEnv => {
 }
 
 export const getAppEnv = (): AppEnv => {
-    const {
+    let {
         app: { domain, emailfrom },
     } = functions.config()
     if (!domain || !emailfrom) {
         throw new InternalServerError('Missing app domain or emailfrom env')
     }
+
+    if (isUsingEmulator()) {
+        domain = 'localhost:3000'
+    }
+
+    const protocol = isUsingEmulator() ? 'http' : 'https'
```

Comment:
.runtimeconfig contain local env, this protocol could be added there as well as setting the localhost domain:port.

Reply:
this way wouldn't we need a separate `.runtimeConfig` file for `dev` & `local` environments ?